src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -324,7 +68,19 @@ def initTest(self):
 
         self.tests = list(self.ExceptionTests.keys())+list(self.SpecialTests.keys())
         self.yangDir = './yang-models/'
-        self.jsonFile = './tests/yang_model_tests/yangTest.json'
+        self.testFile = {}
+
+        for test_file in glob("./tests/yang_model_tests/tests_config/*.json"):
+            try:
+                with open(test_file) as fp:
+                    log.info("Loading file " + test_file)
+                    data = json.load(fp)
```

Comment:
Everytime data is reassigned, the memory referenced by it is going out of scope and should be freed.

Reply:
To keep a Note for this: There are multiple references about json.load being slow, and with many long files here, it will be a concern sooner or later. So we would like to improve it in the future!!!!.  We are not fixing this concern in the current PR.